test/test-applications/integrations/TestApplication.StackExchangeRedis/Program.cs

```diff
@@ -57,8 +56,7 @@ public static async Task Main(string[] args)
 
             db.Ping();
         }
-
-        // SentinelConnect and SentinelConnectAsync introduced in 2.1.50
+#if !REDIS_2_1_50_OR_LOWER
```

Comment:
nit: Following the convention for .NET compiler directives, I prefer renaming this to `REDIS_2_1_50_OR_GREATER` so the directives seem like they're adding code paths for newer versions

Reply:
Also in this case the code wouldn't run if the library version was 2.1.50, even though I think it should

Reply:
Thanks!
Fixed f5c61a756afe85e9f76bed6c2e8d085d64103502